docs/source/en/main_classes/backbones.md

```diff
@@ -0,0 +1,72 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Backbones
+
+Backbones are models used for feature extraction for computer vision tasks. One can use a model as backbone in two ways:
+
+* initializing `AutoBackbone` class with a pretrained model,
+* initializing a supported backbone configuration and passing it to the model architecture. 
+
+## Using AutoBackbone 
+
+You can use `AutoBackbone` class to initialize a model as a backbone and get the feature maps for any stage. You can define `out_indices` to indicate the index of the layers which you would like to get the feature maps from. You can also use `out_features` if you know the name of the layers. You can use them interchangeably. If you are using both `out_indices` and `out_features`, ensure they are consistent. Not passing any of the feature map arguments will make the backbone yield the feature maps of the last layer.
```

Comment:
".. get the feature maps for any stage." - any stage of what? It's a bit unclear.

Reply:
It would be cool to have an illustration here for your specific example. when the `out_indices=(0,)` what exactly does this mean?  It would make it much easier to understand the concept.